sidecar/reloader/main.go

```diff
@@ -22,59 +22,15 @@ const (
 	host = "localhost"
 )
 
-var fileCheck = make(map[string][]byte)
-
-var pluginConfigList = []string{
-	"emqx_auth_http.conf",
-	"emqx_auth_jwt.conf",
-	"emqx_auth_ldap.conf",
-	"emqx_auth_mnesia.conf",
-	"emqx_auth_mongo.conf",
-	"emqx_auth_mysql.conf",
-	"emqx_auth_pgsql.conf",
-	"emqx_auth_redis.conf",
-	"emqx_backend_cassa.conf",
-	"emqx_backend_dynamo.conf",
-	"emqx_backend_influxdb.conf",
-	"emqx_backend_mongo.conf",
-	"emqx_backend_mysql.conf",
-	"emqx_backend_opentsdb.conf",
-	"emqx_backend_pgsql.conf",
-	"emqx_backend_redis.conf",
-	"emqx_backend_timescale.conf",
-	"emqx_bridge_kafka.conf",
-	"emqx_bridge_mqtt.conf",
-	"emqx_bridge_pulsar.conf",
-	"emqx_bridge_rabbit.conf",
-	"emqx_bridge_rocket.conf",
-	"emqx_coap.conf",
-	"emqx_conf.conf",
-	"emqx_dashboard.conf",
-	"emqx_exhook.conf",
-	"emqx_exproto.conf",
-	"emqx_gbt32960.conf",
-	"emqx_jt808.conf",
-	"emqx_lua_hook.conf",
-	"emqx_lwm2m.conf",
-	// "emqx_management.conf",
-	"emqx_modules.conf",
-	"emqx_prometheus.conf",
-	"emqx_psk_file.conf",
-	"emqx_recon.conf",
-	"emqx_retainer.conf",
-	"emqx_rule_engine.conf",
-	"emqx_sasl.conf",
-	"emqx_schema_registry.conf",
-	"emqx_sn.conf",
-	"emqx_stomp.conf",
-	"emqx_tcp.conf",
-	"emqx_web_hook.conf",
+var blackreloadFiles = []string{
+	"emqx_management.conf",
```

Comment:
emqx_dashboard.conf

Reply:
pluginConfigList  didn't shield dashboard

Reply:
`emqx_dashboard` can not be updated by EmqxPlugin Controller, so I think it should be in `blackReloadList`

Reply:
Done